src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/Invitation.cs

```diff
@@ -147,6 +151,11 @@ public void AddParticipant(Participant participant)
                 throw new ArgumentException($"Can't relate item in {participant.Plant} to item in {Plant}");
             }
 
+            if (_participants.Contains(participant))
```

Comment:
What is the motivation here? Is it to prohibit that the very same participant-object is not added twice?
Or is it to prohibit that two different participant-object, which represent the same end user is not added twice?

Reply:
To avoid that the same object is added twice.